### DISCUSSION TITLE
Embedd mermaid generated graph into md

### DISCUSSION THREAD

**Discussion by maica1** [haidaraM/ansible-playbook-grapher#180] [General]
Hello there,
I'm trying to use ansible-playbook-grapher here and the idea of generating code that can be imbedded on markdown is amazing. 
I'm struggling to embbed the contents of the .mmd generated into my readmes, I am able to manually create and visualize some simple graphs on gitlab, but unfortunately copy and pasting the contents of .mmd is not working. 

The graph is not being rendered inside of the ```mermaid``` block, it just outputs the text.
Has anyone here accomplished this?

**Comment by haidaraM** [comment]
Hi,

If I understand correctly some mermaid graphs work on gitlab but not the ones generated by the grapher? 

I haven't tried it yet on Gitlab but will take a look!

**Reply by maica1** [reply]
Yes, simple stuff like below works:
```mermaid

flowchart TD
A(first node) --> B(second node)
```

But when I try the code generated by ansible-playbook-grapher it doesn't render.

Just tried pasting the mermaid code from the playbook-grapher on readme, it didn't rendered as well.
I'm using on prem gitlab 15.8.3-ee,

**Reply by maica1** [reply]
Just tried pasting your sample into gitlab.com and it worked, when I try it on prem it gives the following error:
```
Syntax error in graph
mermaid version 9.1.3
```

I'll try to find the limitation and bring it back here

**Comment by haidaraM** [comment]
I am closing this discussion as I believe this is just a versioning issue on Gitlab's side. Feel free to comment if it's not the case.
